### PR TITLE
fix:added two new fields in reports

### PR DESCRIPTION
**Pull request by siddharth-singh-1**
Added two fields in ach_transfer object for the new flow of simplify report closure

### CODE REVIEW

**Review by KirtiGautam — APPROVED** [review]
LGTM

File: src/components/schemas/fields.yaml [hunk 1]
```diff
@@ -1219,6 +1219,19 @@ ach_transfer:
         - $ref: '#/null_field'
       description: |
         signifies when the ach payment was initiated.
+    source_individual_ach_ref_code:
```
Comment (kartikeyrajvaidya) [inline src/components/schemas/fields.yaml:1222]: `source_account_ach_reference_id`

File: src/components/schemas/fields.yaml [hunk 2]
```diff
@@ -1219,6 +1219,19 @@ ach_transfer:
...
+      nullable: true
+      type: string
+      description: | 
+       Signifies the source of ach transfer reference code.
+      example: IMJYXG4G
+    destination_individual_ach_ref_code:
```
Comment (kartikeyrajvaidya) [inline src/components/schemas/fields.yaml:1228]: destination_account_ach_reference_id

**Review by KirtiGautam — APPROVED** [review]
Looks good

**Pull request merged**